patterns/organisms/components/slider/slider.twig

```diff
@@ -0,0 +1,16 @@
+<div class="{{slider.class}}">
+
+  <div class="{{slider.buttons.next.class}}">{{slider.buttons.next.text}}</div>
+  
+  <div class="{{slider.container.class}}">
+  
+    {% for slide in slider.slides %}
+      {% set promo = slide %}
```

Comment:
I'm fine with this, but maybe there is "prettier" way to pass value to include? maybe this will work `{% include 'molecules/components/promo/promo.twig' with {'promo': slide} %}`, from [docs](http://twig.sensiolabs.org/doc/tags/include.html)